firebase-database/src/main/java/com/google/firebase/database/InternalHelpers.java

```diff
@@ -27,6 +28,7 @@
  *
  * @hide
  */
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
FWIW- I think this whole wrapper class could theoretically be removed in favor of adding RestrictTo to the underlying methods being called.  But I also think it's not worth doing.

Reply:
My helper script is not capable of doing this yet, so I am going to punt on this :)